test/constraint.jl

```diff
@@ -239,6 +239,41 @@ function constraints_test(ModelType::Type{<:JuMP.AbstractModel},
         @test_throws err @constraint(model, [3, x] in SecondOrderCone())
     end
 
+    @testset "Indicator constraint" begin
+        model = ModelType()
+        @variable(model, a, Bin)
+        @variable(model, b, Bin)
+        @variable(model, x)
+        @variable(model, y)
+        for cref in [
+            @constraint(model, a => {x + 2y <= 1}),
+            @constraint(model, a ⇒  {x + 2y ≤ 1})
+        ]
+            c = JuMP.constraint_object(cref)
+            @test c.func == [a, x + 2y]
+            @test c.set == MOI.IndicatorSet{MOI.ACTIVATE_ON_ONE}(MOI.LessThan(1.0))
+        end
+        for cref in [
+            @constraint(model, !b => {2x + y <= 1});
+            @constraint(model, ¬b ⇒  {2x + y ≤ 1});
+            @constraint(model, ![b, b] .=> {[2x + y, 2x + y] .≤ 1});
```

Comment:
Why doesn't this `@constraint` return a vector of `ConstraintRef`s? I'm confused how it's passing tests.

Reply:
I am using `;` so that it concatenates